docs/streams/quickstart.html

```diff
@@ -140,25 +130,36 @@
     --topic streams-file-input
 </pre>
 
+The created topic can be described with the same <b>kafka-topics</b>> tool:
 
 <pre class="brush: bash;">
-&gt; bin/kafka-console-producer.sh --broker-list localhost:9092 --topic streams-file-input < file-input.txt
+&gt; bin/kafka-topics.sh --zookeeper localhost:2181 --describe
```

Comment:
That needs to be done end-to-end in the other quickstarts too IMO. I wasn't even aware the topics tool is depreciated.